cloudflare.go

```diff
@@ -442,7 +442,12 @@ func WithZoneFilters(zoneName, accountID, status string) ReqOption {
 // WithPagination configures the pagination for a response.
 func WithPagination(opts PaginationOptions) ReqOption {
 	return func(opt *reqOption) {
-		opt.params.Set("page", strconv.Itoa(opts.Page))
-		opt.params.Set("per_page", strconv.Itoa(opts.PerPage))
+		if opts.Page != 0 {
```

Comment:
this is great but let's make it a little better by ensuring the value is also positive 

```suggestion
		if opts.Page > 0 {
```